Changelog.md

```diff
@@ -25,6 +25,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Changed display of code declarations in HTML.
   #204 by @mattt.
+- Changed the `generate` command to skip hidden files
+  and top-level `Tests` directories.
```

Comment:
I don't know how surprising this might be for users. I don't expect anyone to try to build documentation for their tests, but maybe we should mention this in the README?

Reply:
That's a great suggestion. Added with a590cad.